code/game/objects/items/cardboard_cutouts.dm

```diff
@@ -179,19 +197,24 @@
 /datum/cardboard_cutout/New()
 	. = ..()
 	if(direct_icon)
-		applied_appearance = mutable_appearance(direct_icon, direct_icon_state)
+		preview_appearance = mutable_appearance(direct_icon, direct_icon_state)
 	else
-		applied_appearance = get_dynamic_human_appearance(outfit, species, mob_spawner, l_hand, r_hand, animated = FALSE)
+		preview_appearance = get_dynamic_human_appearance(outfit, species, mob_spawner, l_hand, r_hand, animated = FALSE)
 
 /// This proc returns the name that the cardboard cutout item will use.
 /datum/cardboard_cutout/proc/get_name()
 	return applied_name
 
 /// This proc sets the cardboard cutout item's vars.
 /datum/cardboard_cutout/proc/apply(obj/item/cardboard_cutout/cutouts)
+	if(isnull(applied_appearance))
+		applied_appearance = image(fcopy_rsc(getFlatIcon(preview_appearance, no_anim = TRUE)))
+	applied_appearance.plane = cutouts.plane
+	applied_appearance.layer = cutouts.layer
 	cutouts.appearance = applied_appearance
 	cutouts.name = get_name()
 	cutouts.desc = applied_desc
+	SEND_SIGNAL(cutouts, COMSIG_CARDBOARD_CUTOUT_APPLY_APPEARANCE, src)
```

Comment:
Don't like this signal, smells like a proc call.
Good news though, it also seems kind of pointless? Why add another signal here when the only place you want to trigger would also be triggered by calling `cutouts.update_icon(NONE)`?